atom/browser/api/atom_api_power_monitor.cc

```diff
@@ -39,6 +60,19 @@ void PowerMonitor::OnResume() {
   Emit("resume");
 }
 
+void PowerMonitor::QuerySystemIdleState(v8::Isolate* isolate, int idle_threshold, const ui::IdleCallback& callback) {
+  if (idle_threshold > 0) {
+    ui::CalculateIdleState(idle_threshold, callback);
+  } else {
+    isolate->ThrowException(v8::Exception::TypeError(
+        mate::StringToV8(isolate, "Invalid idle threshold")));
```

Comment:
Let's be super nice here, "Invalid idle threshold, must be greater than or equal to 0"

Reply:
sounds good, message need to be bit amended to `greater than 0`, having `0` as threshold will not return valid value anyway (while `CalculateIdleState` can accepts it)